integration_tests/tests/forseti/inspec.yml

```diff
@@ -51,9 +51,9 @@ attributes:
 - name: forseti-server-service-account
   required: true
   type: string
-- name: forseti-cai-storage-bucket
-  required: true
-  type: string
+#- name: forseti-cai-storage-bucket
+#  required: true
+#  type: string
```

Comment:
Does this need to be un-commented or removed?

Reply:
Similar to the other one, just uncommenting for now because it can throw an error depending how the tests are run. Will add this into the main outputs and uncomment.